src/Microsoft.AspNet.Mvc.TagHelpers/FormTagHelper.cs

```diff
@@ -124,6 +125,14 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             }
 
             var antiforgeryDefault = true;
+            if (output.Attributes.ContainsName("method"))
```

Comment:
Hmm, we could bind `method`. That way we can avoid all of this tostringing madness below. That way we can just pass it through to the HtmlGenerator